tests/test_renderers/fixtures/reporter_warnings.md

```diff
@@ -110,14 +110,28 @@ Non-consecutive headings:
 <string>:2: (WARNING/2) Non-consecutive header level increase; H1 to H3 [myst.header]
 .
 
-multiple footnote definitions
+footnote reference with no definition
+.
+[^a]
+.
+<string>:1: (WARNING/2) No footnote definition found for label: 'a' [myst.footnote]
+.
```

Comment:
this produced additional docutils warnings, before the fix (https://github.com/executablebooks/MyST-Parser/pull/931/files#r1598103576)